src/Controls/src/Core/ViewExtensions.cs

```diff
@@ -274,6 +281,40 @@ public static Task<bool> TranslateTo(this VisualElement view, double x, double y
 
 			return tcs.Task;
 		}
+		/// <summary>
+		/// Animates an elements <see cref="VisualElement.TranslationX"/> property
+		/// from their current value to the new value. This ensures that the input layout is in the same position as the visual layout.
+		/// </summary>
+		/// <param name="view">The view on which this method operates.</param>
+		/// <param name="x">The x component of the final translation vector.</param>
+		/// <param name="length">The time, in milliseconds, over which to animate the transition. The default is 250.</param>
+		/// <param name="easing">The easing function to use for the animation.</param>
+		/// <exception cref="ArgumentNullException">Thrown when <paramref name="view"/> is <see langword="null"/>.</exception>
+		public static Task<bool> TranslateXTo(this VisualElement view, double x, uint length = 250, Easing easing = null)
+		{
+			if (view == null)
+				throw new ArgumentNullException(nameof(view));
+
+			// Note: We use the same param 'name' as 'TranslateTo' stopping previous 'TranslateTo' animations
+			return AnimateTo(view, view.TranslationX, x, nameof(TranslateTo), (v, value) => v.TranslationX = value, length, easing);
+		}
+		/// <summary>
+		/// Animates an elements <see cref="VisualElement.TranslationY"/> property
+		/// from their current value to the new value. This ensures that the input layout is in the same position as the visual layout.
+		/// </summary>
+		/// <param name="view">The view on which this method operates.</param>
+		/// <param name="y">The y component of the final translation vector.</param>
+		/// <param name="length">The time, in milliseconds, over which to animate the transition. The default is 250.</param>
+		/// <param name="easing">The easing function to use for the animation.</param>
+		/// <exception cref="ArgumentNullException">Thrown when <paramref name="view"/> is <see langword="null"/>.</exception>
+		public static Task<bool> TranslateYTo(this VisualElement view, double y, uint length = 250, Easing easing = null)
+		{
+			if (view == null)
```

Comment:
Hi Marty.

I use 'view == null' instead of 'view is null' because all the other animation methods (FadeTo, LayoutTo, RotateTo...) use that convention in this same file.

Reply:
but you can do the new way, that is just because of old code, you might as well as update those to be consistent to `is null`